waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -572,6 +543,42 @@ when defined(rlnzerokit):
     var rootValue = cast[ptr MerkleNode] (root.`ptr`)[]
     return ok(rootValue)
 
+
+proc updateValidRootQueue*(wakuRlnRelay: WakuRLNRelay, root: MerkleNode): void =
+  # updates the valid merkle root queue with the latest root, and pops the oldest one
+  if wakuRlnRelay.validMerkleRoots.len() == AcceptableRootWindowSize:
```

Comment:
Oh, I see I left a comment on an outdated version. I think using `>=` is wise here to prevent any future memory leaks. If someone were to introduce a minor bug such as adding to `validMerkleRoots` from a different location, this may skip the `==` point and the structure could keep on growing (likely a more serious bug).

Reply:
Addressed in [cf3c2fc](https://github.com/status-im/nwaku/pull/1177/commits/cf3c2fc0379f59e827a8d1ed539cd90d6c2642c4)